client-src/elements/chromedash-form-field.ts

```diff
@@ -466,6 +466,20 @@ export class ChromedashFormField extends LitElement {
             </tr>
           `
         : nothing}
+      ${this.name === 'web_feature' && this.value
+        ? html`
+            <tr>
+              <td colspan="2" class="webdx">
+                <a
+                  href="https://webstatus.dev/features/${this.value}"
+                  target="_blank"
+                >
+                  See feature ${this.value} in Web Platform Status
```

Comment:
The feature name might blend in to this sentence.  E.g., in your PR description screenshot it shows "See feature alerts in Web Platform Status".  While it is true that the user must have selected or typed "alerts" just before that moment, the screenshot could be more clear out-of-context if the name of the feature was visually differentiated.  The cleanest way to do that might be to only link the feature name rather than the whole sentence.

Also, it might be better to refer to it as a "web feature" here.  And maybe refer to the other site as "webstatus.dev" to avoid confusion over the similar site names "Chrome Platform Status" and "Web Platform Status".

Reply:
Done! Thanks for the suggestion